apps/status-api/__tests__/controllers/OracleStatusController.test.ts

```diff
@@ -0,0 +1,87 @@
+import { StatusApiTesting } from '../../testing/StatusApiTesting'
+import { ApiPagedResponse, WhaleApiClient } from '@defichain/whale-api-client'
+import { Oracle, OraclePriceFeed } from '@defichain/whale-api-client/dist/api/Oracles'
+
+const apiTesting = StatusApiTesting.create()
+
+beforeAll(async () => {
+  await apiTesting.start()
+  jest.spyOn(apiTesting.app.get(WhaleApiClient).oracles, 'getOracleByAddress')
+    .mockReturnValue(getMockedOracle())
+})
+
+afterAll(async () => {
+  await apiTesting.stop()
+})
+
+describe('OracleStatusController - Status test', () => {
+  it('/oracles/<address> - should get operational as last published < 45 mins ago', async () => {
```

Comment:
I think should be <=, correct me if I am wrong

```suggestion
  it('/oracles/<address> - should get operational as last published <= 45 mins ago', async () => {
```

Reply:
I judge by reading 
https://github.com/DeFiCh/jellyfish/pull/1354/files#diff-ed767d71dbe26eedb7108dbbe4247fa88d4f5a9c6a7313a637592bd6c23cb049R33